source/constructs/lib/agent/split-job/split_job.py

```diff
@@ -48,19 +48,20 @@ def lambda_handler(event, context):
         job_item["TableBegin"] = str(-1)
         job_item["TableEnd"] = str(-1)
         job_items.append(job_item)
-        event["JobItems"]=job_items
-        return event
-        
-    table_count = get_table_count(event['GlueDatabaseName'], base_time)
+        return job_items
+    
+    glue_database_name = event['GlueDatabaseName']
+    if event.get("IsUnstructured") == 'true':
+        glue_database_name = event['UnstructuredDatabaseName']
+    logger.info(f"glue_database_name:{glue_database_name}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You have a log statement that might use unsanitized input. Depending on the context, this could result in:

1. A log injection attack that breaks log integrity, forges log entries, or bypasses monitors that use the logs. To increase the security of your code, sanitize your inputs before logging them. [Learn more](https://cwe.mitre.org/data/definitions/117.html)

2. A sensitive information leak that exposes users&#39; credentials, private information, or identifying information to an attacker. To preserve privacy in your code, redact sensitive user information before logging it. [Learn more](https://cwe.mitre.org/data/definitions/532.html)